app/src/main/java/com/raulh82vlc/functionalkotlinarrow/data/cache/CacheDataSourceImpl.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2018 Raul Hernandez Lopez
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.raulh82vlc.functionalkotlinarrow.data.cache
+
+import com.raulh82vlc.functionalkotlinarrow.data.cache.model.FeedItemCacheModel
+import java.util.ArrayList
+import java.util.HashMap
+
+/**
+ * Cache data source implementation {@link CacheDataSource} where the repo has its data saved
+ *
+ * @author Raul Hernandez Lopez
+ */
+object CacheDataSourceImpl : CacheDataSource {
```

Comment:
Using the `Impl` indicates that probably the interface is not needed

Reply:
probably a interface doesn´t do a lot for this, needs an object to keep all from the object as a singleton, that´s for sure. nothing major for this